app/models/king.rb

```diff
@@ -1,2 +1,19 @@
 class King < ChessPiece
+  X_MOVE_FACTOR = 1
+  Y_MOVE_FACTOR = 1
+
+  def valid_move?(destination_x, destination_y)
+    fail ArgumentError, 'Invalid Move' if obstructed?(destination_x, destination_y)
+    exceeds_x_movement?(destination_x) || exceeds_y_movement?(destination_y) ? false : true
```

Comment:
This is really nice.

Super nitpick. I think that this could be more readable if it looked like this.

```Ruby
fail ArgumentError, 'Invalid Move' if obstructed?(destination_x, destination_y)

return false if exceeds_x_movement?(destination_x) 
!exceeds_y_movement?(destination_y)
```